### PR TITLE
feat(component): add Image component

### LINKED ISSUE #42: Add component: Image
### Description:

- Add component - `Image`
- Props:
    - src, alt, height, width

### Outcome:

- New `Image` component
- Add PropTypes for the components
- Add Storybook story

### PR DESCRIPTION
**Pull request by niloysikdar**
### This Pull Request covers:

- New component `Image`
- Stories for newly added component
- Props for the component
```
type ImageStyles =
  | 'root'
  | 'table'
  | 'body'
  | 'image'
  | 'caption'
  | 'imageSection'
  | 'captionSection'
  | 'imageColumn';

export interface ImageProps extends BaseStyleProp<ImageStyles> {
  src: string;
  alt: string;
  caption?: string;
  width: CSSProperties['width'];
  height?: CSSProperties['height'];
  captionAlign?: 'left' | 'center' | 'right';
}
```

Resolves #42 

Signed-off-by: Niloy Sikdar <niloysikdar30@gmail.com>

### CODE REVIEW

File: src/components/Image/Image.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        padding: '10px 20px',
+        wordBreak: 'break-word',
+      }}
+    >
+      <img
+        height={height}
+        width={width}
+        src={src}
+        alt={alt}
+        style={{
```
Comment (agentmilindu) [inline src/components/Image/Image.tsx:33]: Let's use `useStyles` from `makeStyles` to combine the styles?

File: src/components/Image/Image.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  className,
+  imageAlign = 'center',
+  captionAlign = 'center',
+}: ImageProps): JSX.Element => {
+  const styles = useStyles({ classes });
+
+  return caption ? (
+    <table cellPadding="0" cellSpacing="0" role="presentation" style={styles.table}>
+      <tbody>
+        <tr>
```
Comment (agentmilindu) [inline src/components/Image/Image.tsx:41]: Let’s add classes to other elements also like `classes.row1`

Comment (niloysikdar) [reply src/components/Image/Image.tsx]: Added classes for other elements following the `Section` and `Column` naming convention.

File: src/components/Image/Image.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+              height={height}
+              width={width}
+              src={src}
+              alt={alt}
+              style={sx(styles.root, styles.image)}
+            />
+          </td>
+        </tr>
+        <tr style={styles.captionSection}>
+          <td style={sx(styles.caption, { textAlign: captionAlign })}>Hello world</td>
```
[the remark this answers is not in the document]
Comment (niloysikdar) [reply src/components/Image/Image.tsx]: Sorry, I missed this part which I did for the testing purpose, now the changes are done.

**Review by agentmilindu — APPROVED** [review]

**Pull request merged**